parse.go

```diff
@@ -581,15 +581,28 @@ func parseOsc(b []byte) (int, Msg) {
 		if len(parts) == 0 {
 			return i, ClipboardMsg("")
 		}
-		b64 := parts[len(parts)-1]
+		if len(parts) != 2 {
```

Comment:
maybe use `strings.Cut`? not sure if it existed in 1.18 or only earlier version

Reply:
But we also want to ensure that we exactly have 2 parts, `strings.Cut` will only split on the first occurrence of `;`.